common/main/cpp/native_c4replicator.cc

```diff
@@ -184,7 +184,7 @@ bool litecore::jni::initC4Replicator(JNIEnv *env) {
 
 static jobject toJavaObject(JNIEnv *env, C4ReplicatorStatus status) {
     jobject obj = env->NewObject(cls_C4ReplStatus, m_C4ReplStatus_init);
-    env->SetIntField(obj, f_C4ReplStatus_activityLevel, (int) status.level);
+    env->SetIntField(obj, f_C4ReplStatus_activityLevel, (int) (status.level == kC4Stopping ? kC4Stopped : status.level));
```

Comment:
Note: Probably better to replace stopping with busy, not stopped.  A lot of CBL consumer code will react to stopped by thinking it is allowed to do things like delete a database, which at this point it would still not be able to do.  Ultimately, core should not be letting this out though!  It takes care to not let it leak via callbacks, but asking for it directly seems leaky!  I'll file an issue to plug that as well.

Reply:
I have another way to fix this that seems much safer to me.  It will introduce a new state in the Java platform, UNKNOWN, which will be used whenever the platform code cannot figure out the replicators state.

Reply:
👍

Reply:
👌 I'll revert this change then. Just fix it elsewhere.